clang/test/dpct/cmake_migration/case_001/expected.txt

```diff
@@ -3,10 +3,7 @@ cmake_minimum_required(VERSION 3.24)
 #CMake_minimum_required(VERSION 3.10)
 project(foo-bar LANGUAGES CXX )
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl")
-find_program(dpct_bin_path NAMES dpct PATHS)
-get_filename_component(bin_path_of_dpct ${dpct_bin_path} DIRECTORY)
-set(dpct_cmake_file_path "${bin_path_of_dpct}/../cmake/dpct.cmake")
-include(${dpct_cmake_file_path})
+include(./dpct.cmake)
```

Comment:
let's use a variable name instead of using "./dpct.cmake" directly.

Reply:
Okay